README.md

```diff
@@ -4,8 +4,8 @@
   the main programming language. 
 * It provides a **reasonably well-written** code example that is **significantly bigger** than what students 
   usually write in data structure modules. 
-* It can be used to achieve a number of beginner-level [learning outcomes](#learning-outcomes) without 
-  running into the complications of OOP or GUI programmings.
+* It can be used to achieve a number of beginner-level learning outcomes without running into the complications
```

Comment:
Perhaps we can change it to link to the new textbook's learning outcomes, unless that is not ready yet? @damithc 

Also a very minor issue: We do not need to maintain the wrapping for documentation, it creates unnecessary noise in the diff.